src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.impl.CsvFileReaderImpl;
+import core.basesyntax.impl.CsvFileWriterImpl;
+import core.basesyntax.impl.FruitImpl;
+import core.basesyntax.impl.ParserImpl;
+import core.basesyntax.impl.ProductDaoImpl;
+import core.basesyntax.impl.ReportServiceImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CsvFileReader;
+import core.basesyntax.service.CsvFileWriter;
+import core.basesyntax.service.Fruit;
+import core.basesyntax.service.Parser;
+import core.basesyntax.service.ProductDao;
+import core.basesyntax.service.ReportService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class Main {
+    private static final String VALID_SOURCE_PATH = "src//main//testFiles//Correct.csv";
+    private static final String INVALID_SOURCE_PATH = "src//main//testFiles//InCorrect.csv";
+    private static final String NEGATIVE_OPERATION_SOURCE_PATH =
+            "src//main//testFiles//NullParameters.csv";
```

Comment:
Place the input and output files into the src/main/resources folder (common mistakes).
```suggestion
    private static final String VALID_SOURCE_PATH = "src//main//testFiles//Correct.csv";
    private static final String INVALID_SOURCE_PATH = "src//main//testFiles//InCorrect.csv";
    private static final String NEGATIVE_OPERATION_SOURCE_PATH =
            "src//main//testFiles//NullParameters.csv";
```